### PR TITLE
change over to styled components

### PR DESCRIPTION
**Pull request by Lsmith2895**
@YelloFattyBean

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,9 +1,6 @@
 <!DOCTYPE html>
 <html>
 <body>
-    <link rel = "stylesheet"
-   type = "text/css"
-   href = "./style.css" />
```
Comment (Lsmith2895) [inline public/index.html:6]: remove link to style.css

File: client/src/components/List.jsx [hunk 2]
```diff
@@ -3,33 +3,70 @@ import React from 'react';
...
+          {product.number_of_ratings}
+          )
+        </div>
+        <div id="related-item-price">
+          $
+          {product.price}
+        </div>
+      </li>
+    );
+  });
```
Comment (Lsmith2895) [inline client/src/components/List.jsx:69]: conditonal render for onsale or sale

**Comment by Lsmith2895** [comment]
@YelloFattyBean

**Review by YelloFattyBean — DISMISSED** [review]

**Review by YelloFattyBean — APPROVED** [review]
Maybe think about moving the styling below the actual function code. Otherwise I approve the code

**Pull request merged**